readme.md

```diff
@@ -359,6 +359,7 @@
 - [ESDoc](https://github.com/esdoc/esdoc) - Documentation generator targeting ES2015, attaching test code and measuring documentation coverage.
 - [Docco](https://github.com/jashkenas/docco) - Documentation generator which produces an HTML document that displays your comments intermingled with your code.
 - [JSDoc](https://github.com/jsdoc3/jsdoc) - API documentation generator similar to JavaDoc or PHPDoc.
+- [Docusaurus](https://github.com/facebook/docusaurus) - Documentation website with Markdown + React.js, easy to maintain
```

Comment:
I think the description is better as "Docusaurus is a performant documentation website generator that leverages on React and Markdown. It comes with translations and versioning features built-in."

Reply:
Almost perfect. I prefer to leave out marketing words like "performant", "simple", etc, as anyone could say that about their project.

Reply:
I create a commit with the description: `Documentation website generator that leverages on React and Markdown. It comes with translations and versioning features built-in.`

Thank you @yangshun and @sindresorhus